airbyte-integrations/connectors/destination-dev-null/build.gradle

```diff
@@ -1,18 +1,26 @@
 plugins {
-    id 'airbyte-java-connector'
+    id 'airbyte-bulk-connector'
 }
 
-airbyteJavaConnector {
-    cdkVersionRequired = '0.20.6'
-    features = ['db-destinations']
-    useLocalCdk = false
+airbyteBulkConnector {
+    core = 'load'
+    toolkits = []
+    cdk = 'local'
 }
 
 application {
     mainClass = 'io.airbyte.integrations.destination.dev_null.DevNullDestination'
-    applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0']
+    //applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0']
+
+    // Uncomment and replace to run locally
+    //applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0', '--add-opens', 'java.base/java.lang=ALL-UNNAMED']
 }
 
+// Uncomment to run locally
+//run {
+//    standardInput = System.in
+//}
+
 dependencies {
-    implementation project(':airbyte-integrations:connectors:destination-e2e-test')
+    testImplementation platform('org.testcontainers:testcontainers-bom:1.19.8')
```

Comment:
this seems very heavy handed

Reply:
I copied that over from the example and misread it as a core dependency; source implementations use it. we don't need it.